bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx64-base.html

```diff
@@ -110,7 +110,7 @@ <h3>{{ sync_head }}</h3>
   ) %}
     <p>
       <a href="https://accounts.firefox.com/signup?service=sync&context=fx_desktop_v3&entrypoint=mozilla.org-fx.wnp64&utm_content=%2Ffirefox%2FWNP%2F64%2F&utm_campaign=wnp64&utm_source=wnp-64" class="mzp-c-button mzp-t-download">{{ button_create }}</a><br>
-      <span class="wn64-signin">{{ login_desc }} <a href="https://accounts.firefox.com/signin">{{ login_link }}</a></span>
+      <span class="wn64-signin">{{ login_desc }} <a href="https://accounts.firefox.com/?action=email&?service=sync&context=fx_desktop_v3&entrypoint=mozilla.org-fx.wnp64&utm_content=%2Ffirefox%2FWNP%2F64%2F&utm_campaign=wnp64&utm_source=wnp-64">{{ login_link }}</a></span>
```

Comment:
after ?action=email, there is `&?`, that extra question mark might cause some issues.